### ISSUE TITLE
Lint command

### ISSUE THREAD

**Issue by Berkmann18** [gridsome/gridsome#326]
## Summary

It would be great if the CLI had a `lint` command like `@vue/cli`, except it might use `eslint-plugin-gridsome` instead of the one for Vue projects.

### Basic example

```bash
gridsome lint
gridsome lint --fix
```

### Motivation

I always have a linter (namely ESLint) which allows me to ensure my code is readable and consistent which ESLint handles well, even on Vue projects where the `lint` service is handy.
It's also quite time-consuming to manually set up ESLint with the correct plugins and configurations in Gridsome projects and I think it would be even better to have it out-of-the-box (which I'm happy to help with).

**Comment by snoopdouglas** [comment]
Any ETA on this?

**Comment by Berkmann18** [comment]
@snoopdouglas Not as far as I'm aware.

**Comment by RehanSaeed** [comment]
It would be great if this also supported https://github.com/remarkjs/remark-lint and https://github.com/davidtheclark/remark-lint-no-dead-urls.

**Comment by Berkmann18** [comment]
@RehanSaeed Yeah, it would be great to have `remark-lint` supported, I haven't used `remark-lint-no-dead-urls` but that seems really useful and nice.

**Comment by rylanharper** [comment]
+1 on this. 👍

In the meantime for those wanted a simple eslint config I read David's article [here](https://dev.to/davidcouronne/gridsome-eslint-the-complete-guide-1fed) 👈 about setting up a simple lint with gridsome. It works great for my projects so far

**Comment by RehanSaeed** [comment]
Prettier and ESLint would also be amazing and is sorely missed.

**Comment by spaceneenja** [comment]
It's pretty easy to add as Rylan indicated above. I didn't think to install @vue/cli-service at first but it makes some sense.

I miss the default @vue/cli commands when working with Gridsome so I installed that too as a dev dependency. I used it to add the eslint plugin.
